standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -3010,3 +3010,5 @@ const string TABLE_BUCKETING_VERSION = "bucketing_version",
 const string DRUID_CONFIG_PREFIX = "druid.",
 const string JDBC_CONFIG_PREFIX = "hive.sql.",
 const string TABLE_IS_CTAS = "created_with_ctas",
+const string PARTITION_TRANSFER_SPEC = "partition_transfer_spec",
```

Comment:
how come this is partition_transfer_spec, unlike elsewhere?

Reply:
I don't know how this happened. Probably I did this typo when I was switching between the IDE and the text editor.